server/src/routers/MenusRouter/getMenuItemList.js

```diff
@@ -0,0 +1,25 @@
+/* 
+*   getMenuItemList(columns, data)
+*
+*   Summary:
+*       Transforms the data returned by the Yale Dining API into a list of MenuItem objects
+*
+*   Parameters:
+*       columns - an array returned by the Yale Dining API describing the organization of the data in data
+*       data - an array of arrays of dining hall data returned from the Yale Dining API, each corresponding to a menu item
+*
+*   Return Value:
+*       an array of MenuItem objects, or null if no items were extracted
+*/
+export default function getMenuItemList(columns, data) {
```

Comment:
Very interesting point. I had assumed the two were fairly interchangable (since `null == undefined`), so didn't give the thing much thought when writing the API reference. On further reflection, it seems like `undefined` is the better bet for behavior here, I'll go back through and change any instance of null to undefined!